packages/terra-docs/src/terra-dev-site/tool/DocumentationComponents/Placeholder.tool.mdx

```diff
@@ -0,0 +1,36 @@
+import { Badge } from '@cerner/terra-docs/package.json?dev-site-package';
+
+import DefaultPlaceholder from '../example/Placeholder?dev-site-example';
+
+import PropsTable from '@cerner/terra-docs/src/Placeholder?dev-site-props-table';
+
+<Badge />
+
+# terra-docs Placeholder
+
+A documentation component intended to be used within documentation examples in place of other child/content elements to clearly denote where the component ends and the placeholder content begins.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install @cerner/terra-docs`
```

Comment:
should it be a dev dep?
```suggestion
  - `npm install @cerner/terra-docs --save-dev`
```

Reply:
If these components are part of your documentation which get exported, it's a true dependency, not a dev dependency.